packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -168,14 +175,24 @@ export class MonthPicker extends React.PureComponent {
       this.props.className
     );
     return (
-      <fieldset className={classes}>
-        {this.renderLabel()}
+      <div className={classes}>
+        <div className="ds-c-label">
+          <h4 className="ds-u-margin--0" id={this.labelId}>
+            {this.props.label}
+          </h4>
+          <p className="ds-u-margin--0 ds-c-field__hint" id={this.hintId}>
+            {this.props.hint}
+          </p>
+        </div>
         <div className="ds-u-margin-y--3">
           {this.renderButton(selectAllText, () => this.handleSelectAll())}
           {this.renderButton(clearAllText, () => this.handleClearAll())}
         </div>
-        <div className="ds-c-month-picker__months">{this.renderMonths()}</div>
-      </fieldset>
+        <fieldset className={classes}>
```

Comment:
Do we need `classes` on this element, as well as the containing `div`? Could we choose one or the other, or split the classes up?

Reply:
I refactored classes and kept the `<fieldset>` dynamic, and hard coded the container.